komodo/deployed.py

```diff
@@ -62,6 +72,7 @@ def deployed_main():
         default=None,
         help="The maximum number of undeployed matrices to list.",
     )
+    parser.add_argument("--json", action="store_true", help="Get output in JSON format")
```

Comment:
Should perhaps add explicit default to CLI, and mention default in Help.

Reply:
The default is automatically set to `False`, but needed to add a `formatter_class` to get the default visible. Fixed.